agent/ebs/watcher.go

```diff
@@ -219,14 +225,12 @@ func (w *EBSWatcher) StageAll(foundVolumes map[string]string) error {
 			&stubFsGroup)
 
 		if err != nil {
-			log.Errorf("Failed to initialize EBS volume: error: %s", err)
-			continue
```

Comment:
we probably want to keep this `continue` here since we don't want to set the status to attached if we encounter an error after staging

Reply:
also, any issues if we keep the log call here as well? that way we don't have to do an additional check if there are any errors after the StageAll call finishes in L.88